server/project-service/src/main/java/edu/illinois/ncsa/incore/service/project/controllers/ProjectController.java

```diff
@@ -425,9 +460,10 @@ public List<DatasetResource> listDatasetsOfProject(
     }
 
     @GET
+    @Deprecated
     @Path("{projectId}/datasets/search")
     @Produces(MediaType.APPLICATION_JSON)
-    @Operation(description = "Search datasets from a project")
+    @Operation(description = "Search datasets from a project", deprecated = true)
```

Comment:
Maybe you can add the message that this method is deprecated, like the above deprecated method?